_includes/head.html

```diff
@@ -0,0 +1,25 @@
+<head>
+  {% if page.title %}
+  <title>{{ site.title }} - {{ page.title }}</title>
+  {% else %}
+  <title>{{ site.title }}</title>
+  {% endif %}
+
+  <meta charset="utf-8" />
+  <meta http-equiv="x-ua-compatible" content="ie=edge" />
+  <title>Vac</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <link
+    rel="shortcut icon"
+    href="{{ '/assets/img/favicon.png' | relative_url }}"
+    type="image/png"
+  />
+
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;1,400;1,600&display=swap"
```

Comment:
Do we need to rely on google fonts? We are quite sensitive about analytics etc in general. Is this a big part of the design or can we make do with something simpler?